ui/app/templates/components/config-pki-ca.hbs

```diff
@@ -4,7 +4,11 @@
     {{#if this.needsConfig}}
       Configure CA Certificate
     {{else}}
-      Replace CA Certificate
+      {{#if this.model.certificate}}
+        Generated Certificate
```

Comment:
Should this be `Generate` as in, the verb?

Reply:
This title shows when the certificate has been generated. Here's a screenshot from binary - so it shows the old `Replace` text. But this is what's been updated to say `Generated`

<img width="735" alt="Screen Shot 2022-05-18 at 9 38 09 AM" src="https://user-images.githubusercontent.com/68122737/169096029-6151b7ad-aa90-4dc8-831e-1815b970c21c.png">

Reply:
It shows up here:

![image](https://user-images.githubusercontent.com/914030/169098018-0fcb43f4-58f3-4006-9337-2266ad83041b.png)

And I think generated is correct :-)